File: src/main/java/com/nayasis/simplelauncher/jpa/repository/LinkRepository.java
package com.nayasis.simplelauncher.jpa.repository;

import com.nayasis.simplelauncher.jpa.entity.LinkEntity;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface LinkRepository extends JpaRepository<LinkEntity,Long> {

    void deleteById( Long id );

    List<LinkEntity> findAllByOrderByGrpAscTitleAsc();

}

File: src/main/kotlin/com/github/nayasis/simplelauncher/jpa/repository/LinkRepository.kt
package com.github.nayasis.simplelauncher.jpa.repository

import com.github.nayasis.simplelauncher.jpa.entity.Link
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface LinkRepository: JpaRepository<Link,Long> {

    fun findAllByOOrderByGroupAndTitle()

}
